spec/watir/rspec/helper_spec.rb

```diff
@@ -7,50 +7,52 @@
   context "#browser" do
     it "delegates to @browser if exists" do
       @browser = :browser
-      browser.should == @browser
+      expect(browser).to be @browser
     end
 
     it "delegates to @browser if @browser exists even if $browser exists" do
       @browser = :browser
       $browser = :global_browser
-      browser.should == @browser
+      expect(browser).to be @browser
     end
 
     it "delegates to $browser if @browser does not exist" do
       $browser = :global_browser
-      browser.should == $browser
+      expect(browser).to be $browser
     end
   end
 
   context "#method_missing" do
     it "redirects missing methods to browser if method exists" do
       @browser = double("browser", coolness_factor: :very)
-      coolness_factor.should == :very
+      expect(coolness_factor).to eql :very
     end
 
     it "raises error when browser does not have method" do
       @browser = double("browser")
-      described_class.should_not be_method_defined :not_existing_method
+      expect(described_class.instance_methods).not_to include :method_to_be_defined
```

Comment:
You've changed here previous behavior of the test by changing that method name. Why did you do that?